### DISCUSSION TITLE
App strategy - a dating-oriented "social network"?

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#6] [Ideas & Features]
Something that came up at the end of an early meeting was that user engagement can be problematic and inconsistent for various reasons. The mostly single-purpose nature of the app for each person (you go there to find dates, or get validation, or find sexual partners, etc.) can make it feel very transactional, shallow, and boring to interact with. This is something I've also given a lot of thought to, and one idea that I've been playing with kind of harkens back to the old school original version of OK Cupid.

When it started, OK Cupid basically had "blogs" that each profile could update, if they wanted to. It also had forums, "references" (you could "vouch" for other people's profiles), and I think some other interesting, more social, less end-game-oriented interaction methods. It was the kind of place where people hung around even after they got a date because they could still help their friends succeed, or maybe they made friends on there with someone half-way across the world and it was the best way they could keep in touch. Or maybe they liked giving dating advice in the forum section. 

The point is it created a *community*, and a whole bunch of other reasons to come back to the app and *add value* for everyone. So I think it's worth considering the possibility of actually aiming for additional features and functionality that allow people to interact beyond the current, simplistic core of dating app engagement: match, message, and that's it. We can do this with strong consideration of the health and wellbeing of users, not with the unethical intend of simply making people addictively use the app more, but to actually find meaningful connection and satisfaction in ways other than finding "the one". This is helpful for everyone, including those still looking for a partner, who (I speak from experience) often just get frustrated and put off of using an app because it's so mechanical and godawful boring.

The other thing this can potentially do is give people spaces and methods to express themselves more naturally, organically, and authentically over time. This in turn lets those who are interested get to know them more naturally. Think about how some people find/found dates on Tumblr, or through Instagram, or other existing creator-oriented social media. There are whole classes of ways to sort of "fall in love" with someone that are completely unserved by current dating app paradigms.

Very curious what you all think of this! I realize that increasing the scope of the app this early on may not be ideal, and fortunately this kind of stuff can always be added incrementally. But I also think it's somewhat representative of a core ethos that we can potentially enact in more ways and from early on: we are building community around dating, around meeting people, and making that process as enjoyable and effective as possible. *Community*, that's really what I'm wanting to create and support, I think. And the app in some ways is just a means to facilitating it, perhaps.

**Comment by aklidata** [comment]
This is the essence of what I was speaking of at the meeting with my Vibecheck ideas and other feature ideas we talked about, if we can make it into a defacto social network it would make it standout a lot.

**Comment by WilliamDormer** [comment]
I love the idea of vouching for people. This was one of the ideas that I was mulling over previously. Have it so that people's friends can verify that they are real, interesting people. This would also decimate the bot population as long as we can verify that the people voting for you are not bots themselves. Maybe linking this with other social media platforms eventually could make this happen? Although I don't think Facebook is gonna be too happy supporting a competing product...

**Reply by aklidata** [reply]
love this idea!

**Comment by ajbott** [comment]
I think the key aspect to this app is going to be the community. The community is what we are really aiming to serve and help support core issues. We'd like people to continue supporting the app, but we'd also like to have a clear direction at first to address current dating app core issues and give people the power to help resolve them.

Implementing a lot of the "social app" features would be a lot of hard work, but if the community wanted it and if it seemed like a viable thing to do, then I wouldn't see any issue with the app evolving to meet these needs. I'd like our mission to connect people together, and in a way, social networking apps do something very similar.
